modules/images-pulling-from-private-registries.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+// * openshift_images/using-image-pull-secrets
+
+[id="images-pulling-from-private-registries_{context}"]
+= Pulling from private registries with delegated authentication
+
+A private registry can delegate authentication to a separate service. In these
+cases, image pull secrets must be defined for both the authentication and
+registry endpoints.
+
+////
```

Comment:
I have this note commented out (and slightly modified) because I don't think "The Red Hat Connect Partner Registry `registry.connect.redhat.com`" exists any more. Is this correct?

If so I'll need to remove the notes in this module *and* from 3.x.